eng/Packages.Data.props

```diff
@@ -138,7 +138,7 @@
     <PackageReference Update="Microsoft.AspNetCore.Http" Version="2.1.22" />
     <PackageReference Update="Microsoft.AspNetCore.Http.Connections" Version="1.0.15" />
     <PackageReference Update="Microsoft.Azure.Functions.Extensions" Version="1.0.0" />
-    <PackageReference Update="Microsoft.Extensions.Azure" Version="1.1.1" />
+    <PackageReference Update="Microsoft.Extensions.Azure" Version="1.1.2" />
```

Comment:
Not needed for Event Hubs, but riding along from today's Extensions package release.